### PR TITLE
Witness styling fix

### PR DESCRIPTION
**Pull request by johnduffell**
image
...becomes... [edit - updated screenshot]
image

### CODE REVIEW

**Comment by stephanfowler** [comment]
:+1:

**Comment by sammorrisdesign** [comment]
Why is the 2nd version in Helvetica rather than our sans?

**Comment by johnduffell** [comment]
not sure tbh, I just reverted to the old styling as far as I know.  I can change things further though!

**Comment by sammorrisdesign** [comment]
Did you take the screenshot on tablet or below? Only reason I can think of as there's no code reason it would have changed.

File: static/src/stylesheets/module/external/_witness.scss [hunk 1]
```diff
@@ -103,14 +103,31 @@
...
     letter-spacing: -.04em;
     text-transform: lowercase;
+
+    .witness__button & {
+        color: #ffffff;
+        font-weight: 700;
+    }
+
+    .element-witness--source & {
+        color: #247eb3;
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/external/_witness.scss]: We should change this `colour(news-support-2)` as it's basically the same and keeps it in the colour family.

**Comment by johnduffell** [comment]
@sammorrisdesign I've tweaked things, I think you're right about the tablet thing as I had dev tools open on the right!

**Pull request merged**